base.js

```diff
@@ -289,5 +289,19 @@ module.exports = {
 
         // LWC import validation
         '@lwc/lwc/no-disallowed-lwc-imports': 'error',
+
+        // Disable any direct importing of LDS artifacts generated by the LWC compiler
+        'no-restricted-imports': [
+            'error',
+            {
+                patterns: [
+                    {
+                        group: ['@salesforce/lds/*'],
```

Comment:
Does this pattern work when doing `import foo from '@salesforce/lds'`? It looks like the extra `/` would prevent that from being covered.